app/views/registrations/edit_v2.html.erb

```diff
@@ -1,4 +1,4 @@
-<% provide(:title, t('registrations.edit_registration.title', person: @user.person.name, comp: @competition.name)) %>
+<% provide(:title, t('registrations.edit_registration.title', person: @user.person&.name || @user.id, comp: @competition.name)) %>
```

Comment:
I think you can use `@user.name` here instead of `@user.id`

Reply:
ah okay makes sense